pdata/internal/semconv/template.j2

```diff
@@ -0,0 +1,87 @@
+{%- macro to_go_attr_type(type, val) -%}
```

Comment:
Is this a copy of template.j2 we have elsewhere?

Reply:
Oh, this was copied by accident. Thanks for catching it